components/network/asterisk/Makefile

```diff
@@ -11,6 +11,7 @@
 #
 # Copyright 2016 Andrey Sokolov. All rights reserved.
 # Copyright 2017 Alexander Pyhalov
+# Copyright 2021 Olaf Bohlen
 #
 
```

Comment:
When you make changes to a package while not upgrading the COMPONENT_VERSION you'll need to set or increment its COMPONENT_REVISION! This is missing here.